pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/ResultHolderFactory.java

```diff
@@ -68,9 +68,11 @@ public static AggregationResultHolder getAggregationResultHolder(AggregationFunc
    *
    * @param function Aggregation function
    * @param capacityCap Max number of results
```

Comment:
I know this is not part of your change, but can we have more explanation on these params, specially given that we are adding more params.